Resources/Prototypes/Datasets/Names/regalrat.yml

```diff
@@ -0,0 +1,39 @@
+- type: dataset
+  id: RegalRatNameKingdom
+  values:
+  - Plague
+  - Miasma
+  - Maintenance
+  - Trash
+  - Garbage
+  - Rat
+  - Vermin
+  - Cheese
+  - Munching
+  - Big
+  - Filth
+  - Sewer
+  - Disposal
+  - Service
+  - The
+
+- type: dataset
+  id: RegalRatNameTitle
+  values:
+  - King
+  - Lord
+  - Prince
+  - Emperor
+  - Supreme
```

Comment:
Wouldn't this be better as an adjective?

Reply:
this is from the tg list. Personally, i think it works pretty well for things like "cheese supreme" and "filth supreme". I don't really see how something like "supreme lord" ties into the whole rat king vibe.